packages/examples/cvat/exchange-oracle/docker-compose.test.yml

```diff
@@ -50,7 +50,7 @@ services:
 
   minio-mc:
     container_name: minio-mc
-    image: minio/mc
+    image: minio/mc:RELEASE.2022-06-10T22-29-12Z
```

Comment:
Latest minio client versions don't have `config` command and tests are failing ([ref](https://github.com/humanprotocol/human-protocol/actions/runs/15285931838/job/42995985118?pr=3364)), so in order to fix that set the version close to the minio itself